internal/cmd/asyncapi/command_import.go

```diff
@@ -0,0 +1,444 @@
+package asyncapi
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/antihax/optional"
+	kafkarestv3 "github.com/confluentinc/ccloud-sdk-go-v2/kafkarest/v3"
+	srsdk "github.com/confluentinc/schema-registry-sdk-go"
+	"github.com/go-yaml/yaml"
+	"github.com/iancoleman/strcase"
+	"github.com/spf13/cobra"
+	spec2 "github.com/swaggest/go-asyncapi/spec-2.4.0"
+	yaml3 "k8s.io/apimachinery/pkg/util/yaml"
+
+	"github.com/confluentinc/cli/internal/pkg/ccloudv2"
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+	"github.com/confluentinc/cli/internal/pkg/errors"
+	"github.com/confluentinc/cli/internal/pkg/examples"
+	"github.com/confluentinc/cli/internal/pkg/kafkarest"
+	"github.com/confluentinc/cli/internal/pkg/log"
+	"github.com/confluentinc/cli/internal/pkg/output"
+)
+
+const parseErrorMessage string = "topic is already present and overwrite flag is false. Moving to the next topic"
+
+type flagsImport struct {
+	overwrite               bool
+	kafkaApiKey             string
+	schemaRegistryApiKey    string
+	schemaRegistryApiSecret string
+}
+
+type BindingsXConfigs struct {
+	CleanupPolicy                  string `yaml:"cleanup.policy"`
+	DeleteRetentionMs              int    `yaml:"delete.retention.ms"`
+	ConfluentValueSchemaValidation string `yaml:"confluent.value.schema.validation"`
+}
+
+type KafkaBinding struct {
```

Comment:
As far as I can tell, this struct is only being used inside this package. Can we make it private?
```suggestion
type kafkaBinding struct {
```